java/gui/src/main/java/com/squareup/subzero/SubzeroGui.java

```diff
@@ -117,8 +136,12 @@ private void interactive() throws Exception {
 
         CommandRequest commandRequest = CommandRequest.parseFrom(proto);
 
+        System.out.println("print out commandRequest:");
+        System.out.println(commandRequest.toString());
```

Comment:
is this needed? or does it come from your testing?

Reply:
Good catch. This should be cleaned up.